nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/test/java/org/apache/nifi/services/azure/storage/TestAzureStorageCredentialsControllerServiceLookup.java

```diff
@@ -71,28 +73,32 @@ public void testLookupServiceA() {
         final AzureStorageCredentialsDetails storageCredentialsDetails = lookupService.getStorageCredentialsDetails(attributes);
         assertNotNull(storageCredentialsDetails);
         assertEquals("Account_A", storageCredentialsDetails.getStorageAccountName());
+        assertEquals("accountsuffix.core.windows.net", storageCredentialsDetails.getStorageSuffix());
     }
 
     @Test
     public void testLookupServiceB() {
```

Comment:
What is the default behavior when the suffix is null? Can this test that we're maintaining the current behavior?

Reply:
null default is handled in SDK. So, the current behavior does not get affected.


private static String getDNS(String service, String base) {
        if (base == null) {
            base = DEFAULT_DNS;
        }
        return String.format(DNS_NAME_FORMAT, service, base);
 }